rll/setup_automatic_upgrade.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Add entry in /etc/cron.d/ to daily check and excute an upgrade for rightlink.
+
+# -e will immediatly exit out of script at point of error
+set -e
+
+cron_file='/etc/cron.d/rightlink_upgrade'
+exec_file='/usr/local/bin/rightlink_check_upgrade'
+
+# Grab toggle option to disable
+if [[ "$DISABLE_AUTO_UPGRADE" == 'true' ]]; then
+  if [ -e ${cron_file} ]; then
+    rm -f ${cron_file}
+    echo "Automatic upgrade disabled"
+  else
+    echo "Automatic upgrade never enabled - no action done"
+  fi
+else
+  # If cron file already exists, will recreate it and update cron config with new random times.
+  [ -e ${cron_file} ] && echo "Recreating cron entry"
+
+  # Generate executable script to run by cron
+  cat > ${exec_file} << 'EOF'
+#!/bin/bash
+
+# This file is autogenerated. Do not edit as changes will be overwritten.
+
+source /var/run/rll-secret
+curl --silent --show-error --get --globoff --fail --request POST --header X-RLL-Secret:$RS_RLL_SECRET \
```

Comment:
IMHO this is not the right call to make. you should make an API1.5 call to schedule the bundle. As-is it does all the interactive stuff, which is pointless and spews tons of stuff into the cron log.